__tests__/app.test.js

```diff
@@ -80,6 +80,49 @@ describe("GET /api/topics", () => {
     })
 })
 
+describe.only("GET /api/articles", () => {
+    it("returns an array of objects", () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                expect(Array.isArray(response.body.articles)).toBe(true)
```

Comment:
This assertion becomes redundant if we are able to use a forEach which can only be called on arrays